openml/flows/flow.py

```diff
@@ -84,8 +362,15 @@ def publish(self):
         self : OpenMLFlow
 
         """
-        xml_description = self._generate_flow_xml()
+        # Checking that the name adheres to oml:casual_string
+        match = re.match(oml_cusual_string, self.name)
```

Comment:
Thanks for pointing that out. I actually removed this, it is a relict from when I uploaded the xml string in the post instead of uploading the xml as a file. The first of those didn't output any xml parsing errors. With the correct way of uploading xml files the server outputs useful error messages and this kind of debugging checks are no longer necessary. I will remove those.